pyalgotrading/algobulls/connection.py

```diff
@@ -247,61 +259,105 @@ def get_report(self, strategy_code, trading_type, report_type, render_as_datafra
             print('Report not available yet. Please retry in sometime')
             # return response
 
-    def backtest(self, strategy_code, start_timestamp, end_timestamp, instrument, lots, strategy_parameters, candle_interval, strategy_mode=StrategyMode.INTRADAY):
+    def backtest(self, strategy=None, start=None, end=None, instruments=None, lots=1, parameters=None, candle=None, mode=StrategyMode.INTRADAY, **kwargs):
         """
         Submit a backtesting job for a strategy on the AlgoBulls Platform
 
         Args:
-            strategy_code: Strategy code
-            start_timestamp: Start date/time
-            end_timestamp: End date/time
-            instrument: Instrument key
+            strategy: Strategy code
+            start: Start date/time
+            end: End date/time
+            instruments: Instrument key
             lots: Number of lots of the passed instrument to trade on
-            strategy_parameters: Parameters
-            candle_interval: Candle interval
-            strategy_mode: Intraday or delivery
+            parameters: Parameters
+            candle: Candle interval
+            mode: Intraday or delivery
+
+        Legacy args (will be deprecated in future release):
+            'strategy_code' behaves same 'strategy'
+            'start_timestamp' behaves same 'start'
+            'end_timestamp' behaves same 'end'
+            'instrument' behaves same 'instrumetns'
```

Comment:
spelling mistake; pls run spellcheck and fix all mistakes.

Reply:
done